projects/valtimo/dossier/src/lib/components/dossier-list/dossier-list.component.html

```diff
@@ -74,6 +74,7 @@
     [items]="obs.documentItems"
     [pagination]="pagination"
     paginationIdentifier="dossierList"
+    translationKey="dossier"
```

Comment:
Update this accordingly, after doing the changes before

Reply:
Changed